netket/hilbert.py

```diff
@@ -1,21 +0,0 @@
-from ._C_netket.hilbert import *
-
-
-def Qubit(graph):
```

Comment:
do we still have Qubit?

Reply:
[yes](https://github.com/netket/netket/blob/4b4788420c5d1415c0c9a1621ea596b7de635086/netket/hilbert/qubit.py#L5)
I think those two files where never loaded. We probably forgot them when we deleted the old stuff...